### DISCUSSION TITLE
Use custom containerDefinition image for ECS Fargate

### DISCUSSION THREAD

**Discussion by Dombo** [artilleryio/artillery#2328] [Artillery]
Hi,

In some of my playwright engine tests I want to [exec](https://nodejs.org/api/child_process.html) out to binaries and use them as part of the scenario, catching the output and making assertions against it. I'd like to be able to bundle the binaries I need within the image and schedule this onto ECS Fargate, I see [here](https://github.com/artilleryio/artillery/blob/f449d5dcfbddcad61f102f1d6453aa55dc5f8399/packages/artillery/lib/platform/aws-ecs/legacy/run-cluster.js#L1042-L1043) that I can seemingly override the imageURI for the worker which runs my scenarios. 

Where is that dockerfile defined? I'd like to extend it when layering in my binaries.

Thanks!

**Comment by hassy** [comment]
Hi @Dombo 👋 You can find the Dockerfile in [PR #2344](https://github.com/artilleryio/artillery/pull/2344).
